projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo.py

```diff
@@ -13,7 +13,7 @@
 )
 
 
-class ObjectNaviThorRGBPPOExperimentConfig(
+class ObjectNavThorRGBPPOExperimentConfig(
```

Comment:
I guess it should be `ObjectNavRoboThorRGBPPOExperimentConfig`. I thought we had gone through all of these things, but good that you double-checked 👍

Reply:
Updated to ObjectNavRoboThorRGBPPOExperimentConfig